app/src/main/res/layout/delete_exceptions_button.xml

```diff
@@ -2,17 +2,14 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<Button android:id="@+id/removeAllExceptions"
+<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/removeAllExceptions"
+    style="@style/ThemeIndependentMaterialGreyButton"
     android:layout_width="wrap_content"
+    android:paddingStart="24dp"
+    android:paddingEnd="24dp"
```

Comment:
Nit: put these in `dimens.xml` file. 

It looks like there are going to be lots of `dp` and `sp` measurements, so perhaps it may be a good idea to open an issue to move all these measures into resource files.